src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/Feeders.scala

```diff
@@ -1,118 +1,14 @@
-  var generatedPassword = ""
-  var generatedEmailForCase = ""
-  var orgName = ""
-  var appReferenceName = ""
-  var sequence1 = 0
-  var seq = 1
-
-  val rnd = new Random()
-  val now = LocalDate.now()
-  val patternDay = DateTimeFormatter.ofPattern("dd")
-  val patternMonth = DateTimeFormatter.ofPattern("MM")
-  val patternYear = DateTimeFormatter.ofPattern("yyyy")
-
-  def sequenceValue() =
-    Stream.continually(repeat.toStream).flatten.take(5).toList
-
-  def randomString(length: Int) = {
-    rnd.alphanumeric.filter(_.isLetter).take(length).mkString
-  }
-
-  def nextSeq() : Integer = {
-    seq = seq + 1
-        seq
-    }
-
-  def generatenextNumber() :Integer = {
-    sequence1 = (sequenceValue().iterator.next())
-    sequence1
-  }
-
-  def generateOrganisationName() :String = {
-    orgName = ("pforgdiv-" + randomString(5))
-    orgName
-  }
-
-  def generateEmailAddress() :String = {
-    generatedEmail = (generateOrganisationName() + "_superuser@mailinator.com")
-    generatedEmail
-  }
-
-  def generateUserEmailAddress() :String = {
-    generatedEmail = (generateOrganisationName() + "_user"+"@mailtest.gov.uk")
-    generatedEmail
-  }
-
-  def generateEmailForCase() :String = {
-    generatedEmailForCase = ("exui_case" + randomString(6) + "@mailtest.gov.uk")
-    generatedEmailForCase
-  }
-
-  def generateAppReferenceName() :String = {
-    appReferenceName = ("case ref perftest" + randomString(6))
-    appReferenceName
-  }
-
-  def generatePassword() :String = {
-    generatedPassword = "Pass19word"
-    generatedPassword
-  }
-
-  def getDay(): String = {
-    (1 + rnd.nextInt(28)).toString.format(patternDay).reverse.padTo(2, '0').reverse //pads single-digit dates with a leading zero
-  }
-
-  def getMonth(): String = {
-    (1 + rnd.nextInt(12)).toString.format(patternMonth).reverse.padTo(2, '0').reverse //pads single-digit dates with a leading zero
-  }
-
-  //Dob >= 35 years
-  def getDobYear(): String = {
-    now.minusYears(35 + rnd.nextInt(70)).format(patternYear)
-  }
-  //Dod <= 21 years
-  def getDodYear(): String = {
-    now.minusYears(1 + rnd.nextInt(20)).format(patternYear)
-  }
-
-  val createDynamicDataFeeder = Iterator.continually(Map("generatedEmail" -> ({
-    generateOrganisationName()+"-su@mailtest.gov.uk"
-  }),
-    "orgName" -> ({
-    orgName
-  })
-  ));
-  /*val createDynamicDataFeeder = Iterator.continually(Map("generatedEmail" -> (generatedEmail), "generatedPassword" -> (generatedPassword), "generateOrganisationName" -> (orgName)));
-*/
-
-  val createCaseData = Iterator.continually(Map("caseEmail" -> ({
-    generateEmailForCase()
-  }),"appRef" -> ({
-    generateAppReferenceName()
-  })));
-
-  val createDynamicUserDataFeeder = Iterator.continually(Map("generatedUserEmail" -> ({
-    "-user"
-  })));
-
-  val IACCreateDataFeeder = Iterator.continually(Map("service" -> ({
-    "IACC"
-  }),
-    "SignoutNumber" -> ({
-      "280"
-    })
+  val orgName = "pforgdiv-" + Common.randomString(5)
```

Comment:
Need to confirm if everything will run ok with the feeder file tidied up, may even be worth moving these objects to the Common file instead and getting rid of the feeder file?